guides/cheatsheets/routing.cheatmd

```diff
@@ -0,0 +1,83 @@
+# Routing cheatsheet
+
+> Those need to be declared in the correct router module and scope.
+
+A quick reference to the common routing features' syntax. For an exhaustive overview, refer to the [routing guides](routing-1.html).
```

Comment:
Once you fix the name, this can be:

```suggestion
A quick reference to the common routing features' syntax. For an exhaustive overview, refer to the [routing guides](routing.md).
```